datasetup/build_US_setup.R

```diff
@@ -156,16 +153,19 @@ if (length(config$subpop_setup$geodata) > 0) {
 # manually remove PR
 census_data <- census_data %>% filter(USPS != "PR")
 
-write.csv(file = file.path(outdir, geodata_file), census_data, row.names=FALSE)
-print(paste("Wrote geodata file:", file.path(outdir, geodata_file)))
+
+if(!dir.exists(dirname(config$subpop_setup$geodata))){
+  dir.create(dirname(config$subpop_setup$geodata))}
+write.csv(file = file.path(geodata_file), census_data, row.names=FALSE)
+print(paste("Wrote geodata file:", file.path(geodata_file)))
 
 
 
 
 # MOBILITY DATA (COMMUTER DATA) ------------------------------------------------------------
 
 
-if(state_level & !file.exists(paste0(config$data_path, "/", config$subpop_setup$mobility))){
+if(state_level & !file.exists(paste0("/", config$subpop_setup$mobility))){
```

Comment:
This also should be changed in line 172
You'll have to remove the `paste0('/' ` as well as this will look for the file in a different place. 
ie
`file.exists(config$subpop_setup$mobility)`